assignment3/WordFinderTest.java

```diff
@@ -0,0 +1,42 @@
+package com.katermar.assignment3;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+
+/**
+ * Created by katermar on 6/9/2017.
+ */
+public class WordFinderTest {
+
+    public static void main(String args[]) {
+        char grid[][] = {
+                {'A', 'A', 'R'},
+                {'T', 'C', 'D'}
+        };
+        String dictArray[] = {"CAR", "CARD", "CART", "CAT"};
+//        String prefixArray[] = {"C", "CA", "CAR", "CARD", "CART", "CAT"};

```

Comment:
Usually you do not want to leave commented out code, especially without a `// TODO` comment explaining what it is for. Such code tends to get neglected during maintenance and rots.